scripts/run_pier.sh

```diff
@@ -0,0 +1,222 @@
+set -e
+
+source x.sh
+
+CURRENT_PATH=$(pwd)
+GODUCK_REPO_PATH=~/.goduck
+PIER_CLIENT_FABRIC_VERSION=master
+PIER_CLIENT_ETHEREUM_VERSION=master
+SYSTEM=$(uname -s)
+
+RED='\033[0;31m'
+BLUE='\033[0;34m'
+NC='\033[0m'
+
+function print_blue() {
+  printf "${BLUE}%s${NC}\n" "$1"
+}
+
+function print_red() {
+  printf "${RED}%s${NC}\n" "$1"
+}
+
+function printHelp() {
+  print_blue "Usage:  "
+  echo "  run_pier.sh <OPT>"
+  echo "    <OPT> - one of 'up', 'down', 'restart'"
+  echo "      - 'up' - bring up a new pier"
+  echo "      - 'down' - clear a new pier"
+  echo "    -t <mode> - pier type (default \"fabric\")"
+  echo "    -r <pier_root> - pier repo path (default \".pier_fabric\")"
+  echo "    -b <bitxhub_addr> - bitxhub addr(default \"localhost:60011\")"
+  echo "  run_pier.sh -h (print this message)"
+}
+
+function prepare() {
+  if [ ! -a bin/bitxhub ]; then
+    mkdir -p bin && cd bin
+    if [ "${SYSTEM}" == "Linux" ]; then
+      tar xf pier-linux-amd64.tar.gz
+      export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:${CURRENT_PATH}/bin/
+    elif [ "${SYSTEM}" == "Darwin" ]; then
+      tar xf pier-macos-x86-64.tar.gz
+      install_name_tool -change @rpath/libwasmer.dylib "${CURRENT_PATH}"/bin/libwasmer.dylib "${CURRENT_PATH}"/bin/bitxhub
+    else
+      print_red "Pier does not support the current operating system"
+    fi
+  fi
+
+  cd "${CURRENT_PATH}"/bin
+  if [ ! -f pier ]; then
+    print_red "pier binary is not downloaded, please download pier first"
+    exit 1
+  fi
+
+  if [ "$MODE" == "fabric" ]; then
+    cd "${CURRENT_PATH}"
+    print_blue "===> Generate fabric pier configure"
+    goduck pier config \
+      --mode "relay" \
+      --bitxhub "localhost:60011" \
+      --validators "0xe6f8c9cf6e38bd506fae93b73ee5e80cc8f73667" \
+      --validators "0x8374bb1e41d4a4bb4ac465e74caa37d242825efc" \
+      --validators "0x759801eab44c9a9bbc3e09cb7f1f85ac57298708" \
+      --validators "0xf2d66e2c27e93ff083ee3999acb678a36bb349bb" \
+      --appchain-type "fabric" \
+      --appchain-IP "127.0.0.1" \
+      --target "${PIER_ROOT}"
+
+    mkdir -p "${PIER_ROOT}"/plugins
+
+    # copy appchain crypto-config and modify config.yaml
+    print_blue "===> Copy fabric crypto-config"
+    if [ ! -d "${CURRENT_PATH}"/crypto-config ]; then
+        print_red "crypto-config not found, please start fabric network first"
+        exit 1
+    fi
+    cp -r "${CURRENT_PATH}"/crypto-config "${PIER_ROOT}"/fabric/
+    x_replace "s/10.1.16.48/localhost/g" "${PIER_ROOT}"/fabric/config.yaml
```

Comment:
the replacement is not needed since it was done in another PR

Reply:
Ok, i will fix it.